README.md

```diff
@@ -1,4 +1,91 @@
-# team-awesome-amijs
+# ✨✨✨ TEAM 💫 AWESOME 💫 AMIJS ✨✨✨
 
-ayeee
-ayooo
+A _majestic_ hour registration app.
+
+## ☄️ The purpose of this app
+
+This app is used for registering working hours. As Kanye West once said:
+
+> Work it, make it, do it
+
+## 🐉 The technologies being used
+
+- [Cypress](https://www.cypress.io/)
+- [Jest](https://jestjs.io/)
+- [Next.js](https://nextjs.org/)
```

Comment:
I would order this from most influential to least influential to the app. For example, `Next.js` is a large cornerstone of the app